buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -145,6 +146,7 @@
       "alwaysRun": false,
```

Comment:
I expect to see a publish to blob feed step in this definition.

Reply:
Reviewing what was already there, all blob uploading I saw for this branch was done in the individual legs as noted above.

Let's chat about this tomorrow AM as I obviously can't merge this today regardless.

Reply:
Updated to allow all legs to continue uploading built packages to blob storage, but left (only) the PublishToBlobStorage variable as a place holder to use when transport feeds turn on.

Reply:
I'll synch with @karajas  when I can about where that work is.  Currently there is no publish-to-blob-feed in release/2.0.0

Reply:
nit: Leaving the PublishToBlobStorage variable is nice, but somewhat confusing until it's implemented.  Also, it pre-supposes the implementation for transport feeds is going to follow your mental model if/when it's implemented.  I'd remove it and file an issue assigned to Karthik or JC to make sure it happens.  That's my 2 cents.